doc/api/http.md

```diff
@@ -841,8 +841,25 @@ Note that the socket timeout logic is set up on connection, so
 changing this value only affects *new* connections to the server, not
 any existing connections.
 
-Set to 0 to disable any kind of automatic timeout behavior on incoming
-connections.
+Set to 0 to disable timeout behavior on incoming connections.
+If `server.timeout` is not 0 and [`server.keepAliveTimeout`][] is 0,
+`server.timeout` will be used for keep-alive timeout.
+
+### server.keepAliveTimeout
+<!-- YAML
+added: REPLACEME
+-->
+
+* {Number} Defaults to 5000 (5 seconds).
+
+The number of milliseconds of inactivity after a server has finished writing
```

Comment:
`The number of milliseconds of inactivity the server needs to wait for additional incoming data, after it has finished writing the last response, before a socket will be destroyed.`